src/Handpose/index.js

```diff
@@ -61,6 +61,9 @@ class Handpose extends EventEmitter {
    */
   async predict(inputOr, callback) {
     const input = this.getInput(inputOr);
+    if (!input) {
```

Comment:
Add this in because whenever we instantiate the model without an image, we'll get a call to predict that produces an exception inside of TF. Instead, it may be better just to return an empty array, but on the other hand it silences the issue. Not sure.

Reply:
Option X:
I wonder if it makes sense to be explicit about when predict() is called? So instead of including it within the loadModel() function, regardless if someone uses a video or otherwise, they need would need to call `.predict()` somewhere in their program to start prediction? 

Option Y:
If we don't go for option Y, I think your solution is good. We can add an optional console.warn() that `predict()` should accept an image or video, but none was passed.